<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>나눔바른펜</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>새로운 입금 내역</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>일자</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>내용</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>입금내역</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>세금처리 확인</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>나눔바른펜</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>미처리 영수 내역 보기</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <family>나눔바른펜</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>세금계산서</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="font">
           <font>
            <family>나눔바른펜</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>현금영수증</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QTableWidget" name="tableWidget_2">
          <column>
           <property name="text">
            <string>입금일</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>입금액</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>내용</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>처리방법</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>나눔바른펜</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>현금영수증 처리 내역</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>551</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
